frameworks/AutoGluon/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mxnet
+autogluon==0.0.6
```

Comment:
autogluon==0.0.7 will release very soon, I would recommend updating this PR to support 0.0.7 before merging.

Reply:
great, given the list of improvements you mentioned in your comments, I'll definitely wait.